build.gradle

```
@@ -1,14 +1,23 @@
 plugins {
-    id 'pl.allegro.tech.build.axion-release' version '1.16.1'
+    alias(libs.plugins.axionRelease)
+    alias(libs.plugins.nexusPublish)
+    id 'java'
 }
 
 group 'com.rundeck'
 
 ext.rundeckPluginVersion = '1.2'
+ext.publishName = "Git Plugin ${project.version}"
+ext.githubSlug = 'rundeck-plugins/git-plugin'
 ext.pluginClassNames='com.rundeck.plugin.GitResourceModelFactory,com.rundeck.plugin.GitCloneWorkflowStep,com.rundeck.plugin.GitPushWorkflowStep,com.rundeck.plugin.GitCommitWorkflowStep,com.rundeck.plugin.GitAddWorkflowStep'
 ext.pluginName = 'Git Plugin'
 ext.pluginDescription = 'This is a git plugin (based on Jgit) which contain a Resource model and worflow steps'
 
+java {
+    sourceCompatibility = JavaVersion.VERSION_11
+    targetCompatibility = JavaVersion.VERSION_11
+}
```

need to generate javadoc/sources jars, see https://github.com/rundeck-plugins/slack-incoming-webhook-plugin/blob/main/build.gradle#L17-L21

```suggestion
java {
    sourceCompatibility = JavaVersion.VERSION_11
    targetCompatibility = JavaVersion.VERSION_11
    withJavadocJar()
    withSourcesJar()
}
```